apps/core/tests/test_schemas.py

```diff
@@ -0,0 +1,34 @@
+from utils.graphene.tests import GraphQLTestCase
+
+from dive.consts import LANGUAGES, HAEADERS
```

Comment:
- Typo: `HAEDERS`.
- It's not clear what do these headers belong to(the actual meaning of headers). Maybe we should name them `TABLE_HEADER_LEVELS`?